k8s/ithc/common/ccd/user-profile-api.yaml

```diff
@@ -20,6 +20,8 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/user-profile-api:prod-d4b45a2e
+      environment:
```

Comment:
there is no idam in user-profile. So, please PR.

Reply:
yes well spotted